indigo_app/forms.py

```diff
@@ -108,13 +108,34 @@ def __init__(self, country, locality, *args, **kwargs):
                 "initial": [{
                     "repealed_work": repealed_work,
                     "repealed_date": repealed_work.repealed_date,
+                    "updated_by_user": repealed_work.updated_by_user,
                 } for repealed_work in self.instance.repealed_works.all()]
             }
             if self.is_bound:
                 repeals_made_formset_kwargs["data"] = self.data
 
             self.repeals_made_formset = self.RepealMadeBaseFormSet(**repeals_made_formset_kwargs)
 
+            self.AmendmentsBaseFormSet = formset_factory(AmendmentForm, extra=0, can_delete=True)
+
+            amendments_formset_kwargs = {
+                "form_kwargs": {
+                    "work": self.instance,
+                },
+                "prefix": "amendments",
+                "initial": [{
+                    "amending_work": amendment.amending_work,
+                    "date": amendment.date,
+                    "created_by_user": amendment.created_by_user,
+                    "updated_by_user": amendment.updated_by_user,
```

Comment:
I think created_by_user and updated_by_user can come out? if you just use self.request.user later